### PR TITLE
add a 2023q2 report for pkgbase.live

### PR DESCRIPTION
**Pull request by igalic**
pkgbase.live is back alive.

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+=== PkgBase.live
+
+Links: +
+link:https://alpha.pkgbase.live/[Website (archive.org)] URL: link:https://alpha.pkgbase.live/[] +
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc:4]: I guess archive.org applied to last quarter, not to this one.

File: website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- aarch64
+- armv7
+- i386
+
+You may notice that RISCv64 is gone for now.
+
+The hardware is a powerful VPS in Vultr.
+The server and the jails running build jobs and serving packages are "self-hosting", meaning they were installed and are kept up-to-date with PkgBase.
+
+Because we haven't figured out yet how to configure Vultr's IPv6 in FreeBSD jails, PkgBase.live is not available over IPv6 for now.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc:32]: Please do not use contractions:
s/haven't/have not/

**Review by lsalvadore — CHANGES_REQUESTED** [review]

File: website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc (hunk 1 above)
Comment (igalic) [reply website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc:4]: correct, i missed that

**Review by igalic — COMMENTED** [review]
@lsalvadore's suggestions

**Comment by igalic** [comment]
will rebase / squash later — if necessary

File: website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Links: +
+link:https://alpha.pkgbase.live/[Website] URL: link:https://alpha.pkgbase.live/[] +
+link:https://codeberg.org/pkgbase[Source] URL: link:https://codeberg.org/pkgbase[]
+
+Contact: Mina Galić <freebsd@igalic.co>
+
+PkgBase.live, an unofficial repository for the FreeBSD link:https://wiki.freebsd.org/PkgBase[PkgBase project], is back alive.
+
+As a service, PkgBase.live was inspired by link:https://up.bsd.lv/[], which provided man:freebsd-update[8] for STABLE and CURRENT branches.
+up.bsd.live itself has gone on hiatus, so it was all the more reason to bring PkgBase.live back.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/pkgbase.live.adoc:12]:
```suggestion
up.bsd.live itself has gone on hiatus, so it was all the more reason to bring back PkgBase.live.
```

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**